dev-libs/libpcre2/libpcre2-10.21.recipe

```diff
@@ -0,0 +1,114 @@
+SUMMARY="Perl5 Compatible Regular Expressions"
+DESCRIPTION="The PCRE2 library is a set of functions that implement regular expression \
+pattern matching using the same syntax and semantics as Perl 5. PCRE2 has its \
+own native API, as well as a set of wrapper functions that correspond to the \
+POSIX regular expression API. The PCRE2 library is free, even for building \
+proprietary software.
+PCRE2 is a re-working of the original PCRE library to provide an entirely new \
+API."
+HOMEPAGE="http://www.pcre.org/"
+COPYRIGHT="	1997-2016 University of Cambridge"
+LICENSE="PCRE"
+REVISION="1"
+SOURCE_URI="https://ftp.csx.cam.ac.uk/pub/software/programming/pcre/pcre2-$portVersion.tar.bz2"
+CHECKSUM_SHA256="c66a17509328a7251782691093e75ede7484a203ebc6bed3c08122b092ccd4e0"
+SOURCE_DIR="pcre2-$portVersion"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86 x86_gcc2"
+
+PROVIDES="
+	libpcre2${secondaryArchSuffix} = $portVersion compat >= 10
+	lib:libpcre2_8${secondaryArchSuffix} = $portVersion compat >= 10
+	lib:libpcre2_16${secondaryArchSuffix} = $portVersion compat >= 10
+	lib:libpcre2_32${secondaryArchSuffix} = $portVersion compat >= 10
+	lib:libpcre2_posix${secondaryArchSuffix} = $portVersion compat >= 10
+	cmd:pcre2grep${secondaryArchSuffix}
+	cmd:pcre2test${secondaryArchSuffix}
+	"
+
+REQUIRES="
+	haiku${secondaryArchSuffix}
+	lib:libz${secondaryArchSuffix}
+	lib:libbz2${secondaryArchSuffix}
+	lib:libedit${secondaryArchSuffix}
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libz${secondaryArchSuffix}
+	devel:libbz2${secondaryArchSuffix}
+	devel:libedit${secondaryArchSuffix}
+	"
+
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:automake
+	cmd:gcc${secondaryArchSuffix}
+	cmd:make
+	cmd:ld${secondaryArchSuffix}
+	cmd:libtoolize
```

Comment:
Safer with:
```
	cmd:libtoolize$secondaryArchSuffix
```
BTW, I think it would be better to use **`$secondaryArchSuffix`** instead of **`${secondaryArchSuffix}`** at every place where it is OK. But feel free to ignore this remark if you prefer using **`${secondaryArchSuffix}`**.